src/metabase/task/send_anonymous_stats.clj

```diff
@@ -0,0 +1,44 @@
+(ns metabase.task.send-anonymous-stats
+  "Contains a Metabase task which periodically sends anonymous usage information to the Metabase team."
+  (:require [clojure.tools.logging :as log]
+            (clojurewerkz.quartzite [jobs :as jobs]
+                                    [triggers :as triggers])
+            [clojurewerkz.quartzite.schedule.cron :as cron]
```

Comment:
As above since `triggers` comes after `schedule` I would ungroup these and alphabetize them correctly